salt/client/mixins.py

```diff
@@ -341,40 +343,3 @@ def print_async_event(self, suffix, event):
             event.pop('_stamp')  # remove the timestamp before printing
```

Comment:
problem is here:
1. i dont think we should pop stuff from the payload as it will disappear from the mutable dict
2. if we dont want the _stamp in the ouput, we then have to do elseway (without deepcopy for obious perf reasons)